code/.gitignore

```diff
@@ -32,7 +32,6 @@
 [Rr]elease/
 [Rr]eleases/
 x64/
-x86/
```

Comment:
This would make Github hub not ignore all x86 folders in the project. Instead can you make an exception rule like [this](https://github.com/ShootingKing-AM/PhoneVR/blob/master/code/.gitignore#L41-L42) for the added x86 libs inclusion ?